lib/getPostcssResult.js

```diff
@@ -18,8 +19,14 @@ module.exports = function (stylelint/*: stylelint$internalApi*/)/*: Promise<?Obj
     syntax?: stylelint$syntaxes,
     customSyntax?: string
   }*/ = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {}
+  // eslint-disable-next-line
+  console.log('----------------------------')
+  // eslint-disable-next-line
+  console.log('OPTIONS ==> ', options)
+  // eslint-disable-next-line
+  console.log('===============')
```

Comment:
Please remove these console logs (must be a mistake, right?), then we can merge.